ztools/x509/json.go

```diff
@@ -136,14 +136,16 @@ func (p *PublicKeyAlgorithm) UnmarshalJSON(b []byte) error {
 }
 
 type auxValidity struct {
-	Start string `json:"start"`
-	End   string `json:"end"`
+	Start          string `json:"start"`
+	End            string `json:"end"`
+	ValidityPeriod int    `json:"length"`
```

Comment:
I would call this "hours" or something that implies it's in hours and not seconds.